base_wamas_ubl/models/base_wamas_ubl.py

```diff
@@ -27,3 +28,7 @@ def parse_wamas2ubl(self, str_file, extra_data=False):
     @api.model
     def parse_ubl2wamas(self, str_file, telegram_type):
```

Comment:
```suggestion
    def ubl2wamas(self, str_file, telegram_type):
```

Reply:
let's get rid of confusing `parse_` prefix and just re-use same names as method exposed